management/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.6.7] - 2021-07-07
+
+- Fixed an issue with rollback case in `updateIdentityFromUser`.
+- 💥 Removed an identity from `IdentityTypes` enum.
```

Comment:
if 💥 means `breaking change` it should be a major version bump so
```suggestion
## [0.7.0] - 2021-07-07

- Fixed an issue with rollback case in `updateIdentityFromUser`.
- 💥 Removed an identity from `IdentityTypes` enum.
```

Reply:
The real thing is, is this removal really a breaking change in the end ? Wdyt about that ?

Reply:
It is, because if someone relied on it for any reason, it break its application

Reply:
Except that on Connect side, it is removed since a long time. We just forgot to do it on our side in time. So, this someone should have its application broken since a long time, no ? 
Anyway, I'm making the changes.